app/models/pawn.rb

```diff
@@ -1,21 +1,24 @@
 # Class for Pawn Piece
 class Pawn < Piece
   def valid_move?(x_new, y_new)
-    move_y = move_y(y_new)
-    move_x = move_x(x_new)
     return false if guard_move_is_on_board?(x_new, y_new)
```

Comment:
You don't need to have return false for guard_move_is_on_board?. It doesn't return true or false, it just raises an error if a move off the board is attempted.